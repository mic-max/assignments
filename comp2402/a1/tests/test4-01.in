Car
Car
CAR
car
money
$$$
$
$$$
$$$$
$$$$$
$$
$$
$
donley
donkey
donley
happy
sad
mad
glad
fad
juice
glad
mad
mad
mad
mad