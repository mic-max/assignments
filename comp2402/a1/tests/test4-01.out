Car
$$$
$$
$
donley
glad
mad
mad
mad
mad
